### PR TITLE
DAT-61703 App Components | Pagination Hooks refactoring 

### PR DESCRIPTION
**Pull request by mlevi1806**

### CODE REVIEW

File: app/src/components/base/Pagination.js [hunk 1]
```diff
@@ -105,54 +82,61 @@ export default class Pagination extends React.Component {
...
-          {selected} out of {total}
-        </Info>
-      </Container>
-    );
-  }
-}
+  }, [current, max, selected, setPage, total]);
+
+  useEffect(() => {
+    update();
```
Comment (crazypixel) [inline app/src/components/base/Pagination.js:94]: i see that this is the only use of update. you can just place it here.

Comment (mlevi1806) [reply app/src/components/base/Pagination.js]: @crazypixel Done.

And yes, i did tested the component :)

**Pull request merged**